index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Belo Horizonte/MG - Brazil</title>
+    </head>
+    <body>
+        <header>
+            <h1 style="text-align: center">Welcome to <br> BELO HORIZONTE</h1>
+        </header>
+        <nav>
+            <ul>
+                <a href="#home"><li>Home</li></a>
+                <a href="#about"><li>About</li></a>
+                <a href="#history"><li>History</li></a>
+                <a href="#location"><li>Location</li></a>
+                <a href="#gastronomy"><li>Gastronomy</li></a>
+                <a href="#more"><li>More</li></a>
+                <a href="#contact"><li>Contact</li></a>
+            </ul>
+        </nav>
+        <hr>
+        <main>
+            <section id="about">
+                <h2>About the City</h2>
+                <a href="">Back to the top</a>
```

Comment:
the href is empty? did you forgot to link to the top ;-)

Reply:
I included an id on the header, then the href now links to the top of the page. When it is empty, I think it just reloads the page right?

Reply:
When it's empty it should do nothing.

Reply:
ok